.github/ISSUE_TEMPLATE/airflow_providers_bug_report.yml

```
@@ -66,6 +66,7 @@ body:
         - influxdb
         - jdbc
         - jenkins
+        - kafka
```

```suggestion
        - apache-kafka
```

to follow the same convention as Apache beam, spark etc